projects/demo/src/modules/components/input-date-time/input-date-time.component.ts

```diff
@@ -61,24 +61,24 @@ export class ExampleTuiInputDateTimeComponent extends AbstractExampleTuiControl
         'value-transformer.ts': import('!!raw-loader!./examples/4/value-transformer.ts'),
     };
 
-    readonly minVariants = [
+    readonly minVariants: [TuiDay, TuiDay, TuiDay, [TuiDay, TuiTime]] = [
```

Comment:
How about just `ReadonlyArray<TuiDay | [TuiDay, TuiTime]>`? And in similar spots here.

Reply:
fixed